apps/deploy-web/src/utils/localStorage.ts

```diff
@@ -1,36 +1,26 @@
 "use client";
 
+import getConfig from "next/config";
 import { gt, neq } from "semver";
+
 import { mainnetId } from "./constants";
-import getConfig from "next/config";
 const { publicRuntimeConfig } = getConfig();
 
 const migrations = {
-  // TODO
   "0.14.0": () => {}
 };
 
 // Store latestUpdatedVersion in localStorage
 // Check if latestUpdatedVersion is < currentVersion
 // If so run all the version > until current is reached.
 export const migrateLocalStorage = () => {
-  const currentVersion = publicRuntimeConfig?.version;
-  let latestUpdatedVersion = localStorage.getItem("latestUpdatedVersion");
-
-  if (!latestUpdatedVersion) {
-    // It's an upgrade from an old version
-    if (Object.keys(localStorage).some(key => key.endsWith(".data") || key.endsWith(".wallet"))) {
-      latestUpdatedVersion = "1.0.0";
-    } else {
-      // It's a brand new installation
-      latestUpdatedVersion = currentVersion;
-    }
-  }
+  const currentVersion: string = publicRuntimeConfig.version;
+  const version = getVersion();
+  const hasPreviousVersion = version && neq(currentVersion, version);
 
-  // Only apply migrations if there was a previous version
-  if (latestUpdatedVersion && neq(currentVersion, latestUpdatedVersion)) {
+  if (hasPreviousVersion) {
     Object.keys(migrations).forEach(version => {
-      if (gt(version, latestUpdatedVersion)) {
+      if (gt(version, version)) {
```

Comment:
maybe not the best to both have the same name :)

Reply:
right

Reply:
done